Makefile.am

```diff
@@ -4047,6 +4047,7 @@ selinux_child_CFLAGS = \
     $(NULL)
 selinux_child_LDADD = \
     libsss_debug.la \
+    libsss_util.la \
```

Comment:
selinux_child is suid binary so it should have minimal dependencies.
Please add `src/util/util_errors.c` to SOURCES instead of linking with `libsss_util.la`